From 44bef734096be40e96179cc1ebde9a7bd6e1cd2f Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:49:16 -0400
Subject: [net] svcrpc: remove an unnecessary assignment

Message-id: <1403560314-32525-15-git-send-email-bfields@redhat.com>
Patchwork-id: 84485
O-Subject: [Patch RHEL7 014/172] svcrpc: remove an unnecessary assignment
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Weng Meiling <wengmeiling.weng@huawei.com>

Signed-off-by: Weng Meiling <wengmeiling.weng@huawei.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 587ac5ee6f9f912926abe34bb323c6f87d53f137
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/sunrpc/svc.c b/net/sunrpc/svc.c
index 89a588b..e009aab 100644
--- a/net/sunrpc/svc.c
+++ b/net/sunrpc/svc.c
@@ -1104,8 +1104,6 @@ svc_process_common(struct svc_rqst *rqstp, struct kvec *argv, struct kvec *resv)
  rqstp->rq_vers = vers = svc_getnl(argv); /* version number */
  rqstp->rq_proc = proc = svc_getnl(argv); /* procedure number */
 
- progp = serv->sv_program;
-
  for (progp = serv->sv_program; progp; progp = progp->pg_next)
   if (prog == progp->pg_prog)
    break;
-- 
1.7.1